lab_1/main.py

```diff
@@ -2,24 +2,24 @@
 Language detection
 """
 
+import json
+
+
 def tokenize(text: str) -> list or None:
     """
     Splits a text into tokens, converts the tokens into lowercase,
     removes punctuation and other symbols from words
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    if isinstance(text, str):
-        symbols = """'!@#$%^&*()-_=+/|"№;%:?><,.`~’…—[]{}1234567890\t\n\r"""
-        for i in text:
-            if i in symbols:
-                text = text.replace(i, '')
-            text = text.lower()
-        tokens = text.split()
-        return tokens
-    else:
+    if not isinstance(text, str):
         return None
-    pass
+    puncs = """'!@#$%^&*()-_=+/|"№;%:?><,.`~’…—[]{}1234567890\t"""
+    for symbol in text:
+        if symbol in puncs:
+            text = text.replace(symbol, '')
```

Comment:
and if the symbol is `\t`? 
is that a problem?

Reply:
As far as I understand, it would be a problem if we leave symbol \t in _puncs_, because words separated with tabulation would combine with each other. .split() method should be just fine